portfolio/src/main/webapp/script.js

```diff
@@ -67,17 +66,17 @@ function deleteTask(task) {
   fetch('/data', {method: 'POST', body: params});
 }
 
-google.charts.load('current', {'packages':['corechart']});
+google.charts.load('current', { 'packages':['corechart']});
```

Comment:
Nit: is this suggested by linter?

Reply:
Yes it was, I just have to fix the rest of my code according to the lint check.